Dockerfile

```diff
@@ -1,40 +1,55 @@
-# build
-FROM ubuntu:18.04 AS builder
+FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
```

Comment:
We could build "locally" and just copy the target into the image. In this case, docker image is much smaller.

Reply:
That's true. But I would keep the building inside Docker so someone who can't build Silius locally (due to unsupported architecture) can build it in Docker.